modules/core/src/v2/coins/algo.ts

```diff
@@ -135,28 +198,82 @@ export class Algo extends BaseCoin {
   /**
    * Specifies what key we will need for signing - Algorand needs the backup, bitgo pubs.
    */
-  keyIdsForSigning() {
+  keyIdsForSigning(): number[] {
     return [ 0, 1, 2 ]; 
   }
 
+  /**
+   * Explain/parse transaction
+   * @param params
+   * - txBase64: transaction encoded as base64 string
+   * @returns {{displayOrder: [string,string,string,string,string], id: *, outputs: Array, changeOutputs: Array}}
```

Comment:
This looks odd. Can `displayOrder` type be expressed as `displayOrder: string[]`

Reply:
hmm sure this is the standard applied in the other docs. I'll put displayOrder[6]

Reply:
err string[5]